src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -704,6 +708,47 @@ public boolean isPrivate() {
         return _private;
     }
 
+    /**
+     * Visibility of a repository.
+     */
+    public enum Visibility {
+        PUBLIC, INTERNAL, PRIVATE, UNKNOWN;
+
+        public static Visibility from(String value) {
+            return EnumUtils.getNullableEnumOrDefault(Visibility.class, value, Visibility.UNKNOWN);
+        }
+
+        @Override
+        public String toString() {
+            return name().toLowerCase(Locale.ROOT);
+        }
+    }
+
+    /**
+     * Gets the visibility of the repository.
+     *
+     * @return the visibility
+     */
+    @Deprecated
+    @Preview(NEBULA)
+    @WithBridgeMethods(value = String.class, adapterMethod = "visibilityAsStr")
```

Comment:
```suggestion
```

Bridge methods are only needed to maintain binary compatibility with previously published API.  You're adding new methods, so they are not needed.